lib/main.js

```diff
@@ -7,6 +9,11 @@ var showHotKey = Hotkey({
   }
 });
 
```

Comment:
A compile-time error in the JavaScript halted validation of that file.Message: missing ; before statement